tests/Unit/Numerical/Spectral/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 # See LICENSE.txt for details.
 
 set(SPECTRE_UNIT_SPECTRAL_TESTS
+    Numerical/Spectral/DefiniteIntegral.cpp
     Numerical/Spectral/LegendreGaussLobatto.cpp)
```

Comment:
It'll cause fewer conflicts going forward if the `)` is moved to it's own line (my fault, not yours)